docs/react.md

```diff
@@ -63,7 +73,38 @@ const { channel } = useChannel("your-channel-name", (message) => {
 
 Every time a message is sent to `your-channel-name` it'll be logged to the console. You can do whatever you need to with those messages.
 
-Our react hooks are designed to run on the client-side, so if you are using server-side rendering, make sure that your components which use Ably react hooks are only rendered on the client side.
+> [!NOTE]
+> Our react hooks are designed to run on the client-side, so if you are using server-side rendering, make sure that your components which use Ably react hooks are only rendered on the client side.
+
+### Channel Options
+
+We support providing [ChannelOptions](https://ably.com/docs/api/realtime-sdk/types#channel-options) for the `ChannelProvider` component:
+
+This means you can use features like `rewind`:
+
+```jsx
+<ChannelProvider channelName="your-channel-name" options={{ params: { rewind: '1' } }}>
+  <Component />
+</ChannelProvider>
+```
+
+[Subscription filters](https://ably.com/docs/channels#filter-subscribe) are also supported:
+
+```jsx
+const deriveOptions = { filter: 'headers.email == `"rob.pike@domain.com"` || headers.company == `"domain"`' }
+
+return (
+  <ChannelProvider channelName="your-derived-channel-name" options={{ ... }} deriveOptions={deriveOptions}>
+    <Component />
+  </ChannelProvider>
+)
+```
+
+> [!NOTE]
+> Please note that attempting to publish to a derived channel (one created or retrieved with a filter expression) will result in failure.
```

Comment:
this paragraph now clashes with https://github.com/ably/ably-js/pull/1658.
I suggest we make this PR to be based on https://github.com/ably/ably-js/pull/1658 and figure out the correct wording for this part there.